src/components/EditListLayout/EditListLayout.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, ReactElement } from 'react';
+import { useIntl } from 'react-intl';
+import { isNumber } from 'lodash';
+import { Button, Layer, Loading, Pane, PaneFooter, Paneset } from '@folio/stripes/components';
+import { t } from '../../services';
+import { ListAppIcon } from '../ListAppIcon';
+
+type EditListLayoutProps = {
+  isSaveButtonDisabled?: boolean;
+  isLoading?: boolean;
+  onCancel?: () => void;
+  onSave?: () => void;
+  children?: ReactElement[],
+  name: string,
+  title: string | ReactElement,
+  recordsCount?: number,
+  lastMenu?: ReactElement
+};
+
+export const EditListLayout: FC<EditListLayoutProps> = ({
+  children,
+  name,
+  title,
+  isLoading,
+  recordsCount,
+  onCancel,
+  isSaveButtonDisabled,
+  onSave,
+  lastMenu
+}) => {
+  const { formatNumber } = useIntl();
+  const getRecordCountString = (count: any) => {
+    return isNumber(count) ? t('mainPane.subTitle',
+      { count: formatNumber(count) }) : '';
+  };
+
+  const paneSub = !isLoading ? getRecordCountString(recordsCount)
+    :
+  <>{t('lists.item.loading')}<Loading /></>;
```

Comment:
Super tiny nit: the formatting here is a bit odd, harming readability

Reply:
agreed - i really like the [new prettier ternary](https://prettier.io/blog/2023/11/13/curious-ternaries) if you need inspiration

Reply:
I kind of like that, too... I tend to do
```
result = condition
? trueResult
: falseResult
```
because I like how everything lines up vertically. But I second guess myself every time because `condition ?` makes so much sense on a line, since it looks like a question :D